src/utils/update/managing.officer.mapper.ts

```diff
@@ -88,3 +88,11 @@ export const mapMoPrivateAddress = (moPrivateData: ManagingOfficerPrivateData[],
     }
   }
 };
+
+export const mapMoPrivateEmailAddress = (moPrivateData: ManagingOfficerPrivateData[], ch_reference: string) => {
```

Comment:
Same comment as in your other PR, may be good to combine the MO data mapping into one function (where we have duplicate code)